### ISSUE TITLE
dim 

### ISSUE THREAD

**Issue by laokelaoke** [sjmoran/CURL#37]
In the util file：
    def rgb_to_lab(img, is_training=True):
         img = img.permute(2, 1, 0) 

in the model file:
  img_lab = torch.clamp(ImageProcessing.rgb_to_lab(img_clamped.squeeze(0)), 0, 1)

Error occurred: number of dims don't match in permute

**Comment by sjmoran** [comment]
Check the dimensions of the data you are loading in to see if they are as expected for the code.
